spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/rsocket/RSocketProperties.java

```diff
@@ -121,6 +123,74 @@ public void setSsl(Ssl ssl) {
 			this.ssl = ssl;
 		}
 
+		public Spec getSpec() {
+			return this.spec;
+		}
+
+		public void setSpec(Spec spec) {
+			this.spec = spec;
+		}
+
+		public static class Spec {
+
+			/**
+			 * Sub-protocol to use in websocket handshake signature. Null by default.
+			 */
+			private String protocols;
+
+			/**
+			 * Specifies a custom maximum allowable frame payload length. 65536 by
+			 * default.
+			 */
+			private int maxFramePayloadLength;
```

Comment:
This field should be initialized  with `65536` then. Currently the advertised default is `0`.